src/shared/docs/en/guides/quickstart.md

```diff
@@ -17,27 +17,27 @@ While this guide assumes general familiarity with such things as text editors, t
 ### tl;dr, you'll need:
 - A (free) GitHub account
 - Any computer running macOS, Windows, or Linux, with:
-  - Node.js 12
-  - npm 6.x
+  - Node.js 12, or later
+  - npm 6, or later
 
 
 ### Platforms: macOS, Windows, and Linux
 
 Begin supports local, offline development on just about all modern computers, including: macOS, Windows (10+), and Linux.
 
 
-### Runtimes: Node.js 12
+### Runtimes: Node.js
 
-Begin provisions and manages AWS infrastructure that runs solely on Node.js 12 (with support additional runtimes coming soon).
+Begin provisions and manages AWS infrastructure that runs on Node.js (with support for additional runtimes coming soon). Node.js 12.x is the default runtime, but you can [configure](https://arc.codes/docs/en/reference/project-manifest/aws#runtime) Node.js 14.x to be the runtime on a per-project or per-function level.
```

Comment:
I think that node 14 is now the default.

Reply:
@ryanbethel can you try deploying a new app and checking the node version number? It seems to me that it is still node 12.